package.json

```diff
@@ -34,7 +34,13 @@
     "eslint-plugin-react": "^7.4.0",
     "jest": "^21.1.0",
     "react": "^15.6.2",
+    "react-addons-test-utils": "^15.4.2",
```

Comment:
Are these changes necessary?

Reply:
Set it up for testing. For TravisCI to run tests on this repo by itself, it is needed.